### DISCUSSION TITLE
Take this app into production as an actual Shopify custom app

### DISCUSSION THREAD

**Discussion by AminRafaey** [kinngh/shopify-node-mongodb-next-app#43] [Q&A]
Hi @kinngh thank you so much for such an amazing repo. It taught me and I could say to a lot of people how to build Shopify apps. But how can I take the app I built using this repo into production now? So I could ask anyone hey man install this app. What steps would you suggest for beginners like me who just created their very first app using this repo?

**Comment by kinngh** [comment]
Thank you for the kind words! Use a database provider of your choice, host the app on Vercel and fill up the App Store listing and submit it for review! 

https://shopify.dev/docs/apps/store

**Reply by AminRafaey** [reply]
Thank you @kinngh, Is there any possible way that until my app doesn't get verified from Shopify, I can ask people to install that app just for testing purposes? I know I can do this by following an approach where we add secrets in .env then log in over Shopify and update URLs by using `update:url`. You can feel you can't ask someone to do that all so is there any simple way for someone else to test your app which is in development right now?

**Reply by kinngh** [reply]
Create a single install link and share it with your merchants. Trying to circumvent Shopify's ToS will mostly get your Partner account suspended, so it makes more sense to follow what's recommended instead of building workarounds
